From c90071e2caaf28da14536e4b3f6c74cba71c924c Mon Sep 17 00:00:00 2001
From: Chrome Release Autoroll
 <chromium-release-autoroll@skia-public.iam.gserviceaccount.com>
Date: Thu, 25 Jun 2020 09:52:48 +0000
Subject: [PATCH 04/12] Roll ChromeOS Broadwell AFDO profile from
 83-4103.97-1592218687-benchmark-83.0.4103.117-r1 to
 83-4103.97-1592819197-benchmark-83.0.4103.119-r1

If this roll has caused a breakage, revert this CL and stop the roller
using the controls here:
https://autoroll.skia.org/r/cros-afdo-broadwell-chromium-stable
Please CC c-compiler-chrome@google.com on the revert to ensure that a human
is aware of the problem.

To report a problem with the AutoRoller itself, please file a bug:
https://bugs.chromium.org/p/skia/issues/entry?template=Autoroller+Bug

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+doc/master/autoroll/README.md

Tbr: c-compiler-chrome@google.com
Change-Id: I3de1bd68eda26a92502c2e38044b2bd4dd855d71
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2266019
Reviewed-by: Chrome Release Autoroll <chromium-release-autoroll@skia-public.iam.gserviceaccount.com>
Commit-Queue: Chrome Release Autoroll <chromium-release-autoroll@skia-public.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/4103@{#727}
Cr-Branched-From: 8ad47e8d21f6866e4a37f47d83a860d41debf514-refs/heads/master@{#756066}
---
 chromeos/profiles/broadwell.afdo.newest.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chromeos/profiles/broadwell.afdo.newest.txt b/chromeos/profiles/broadwell.afdo.newest.txt
index 7ca64107ca9..953aba58a64 100644
--- a/chromeos/profiles/broadwell.afdo.newest.txt
+++ b/chromeos/profiles/broadwell.afdo.newest.txt
@@ -1 +1 @@
-chromeos-chrome-amd64-broadwell-83-4103.97-1592218687-benchmark-83.0.4103.117-r1-redacted.afdo.xz
+chromeos-chrome-amd64-broadwell-83-4103.97-1592819197-benchmark-83.0.4103.119-r1-redacted.afdo.xz
-- 
2.28.0

